Include/Acidanthera/Library/OcConfigurationLib.h

```diff
@@ -448,6 +448,7 @@ typedef enum {
   _(OC_STRING                   , SystemUuid         ,     , OC_STRING_CONSTR ("", _, __)                 , OC_DESTR (OC_STRING) ) \
   _(OC_STRING                   , Mlb                ,     , OC_STRING_CONSTR ("OPENCORE_MLB_SN11", _, __), OC_DESTR (OC_STRING) ) \
   _(OC_STRING                   , SystemMemoryStatus ,     , OC_STRING_CONSTR ("Auto", _, __)             , OC_DESTR (OC_STRING) ) \
+  _(OC_STRING                   , BIOSVersion        ,     , OC_STRING_CONSTR ("", _, __)                 , OC_DESTR (OC_STRING) ) \
```

Comment:
Should be
`_(OC_STRING                   , BIOSVersion        ,     , OC_STRING_CONSTR ("Auto", _, __)                 , OC_DESTR (OC_STRING) ) \`